lib/SlackBot.js

```diff
@@ -4,6 +4,7 @@ var express = require('express');
 var bodyParser = require('body-parser');
 var querystring = require('querystring');
 var async = require('async');
+var Cryptr = require('cryptr');
```

Comment:
@abratnap For what purpose do you need to encrypt the team and user tokens? I get the basic idea (security measures), but what if you would like to read those data manually, lets say for checking reasons what team has connected to your app? Do you provide a decrypt function to establish connection of your bot to each connected teams?